### ISSUE TITLE
Output string from "in"

### ISSUE THREAD

**Issue by dome2048** [QNimbus/node-red-contrib-openhab-v2#23]

**Comment by dome2048** [comment]
I was probably too tired to be trying to do this yesterday... I did figure out how to connect the nodes. For the benefit of anyone who comes after me, I used a change node and set the output to msg.payload.state.

I'm keeping the issue open as it still would be nice to have an option to output just the state without a change node.

I forgot to include this info yesterday: I'm running an Ubuntu 20.04.1 server version with nodeRED 1.1.3 in a Docker container ( as is openHAB).

**Comment by QNimbus** [comment]
@dome2048 You just beat me to it, but that is indeed the way to access the 'state'. 

I will mark this issue as a feature request though, that ks for your support!

**Comment by dome2048** [comment]
Absolutely, thanks for your work on this plugin! So far it looks great!
